src/custom/components/OrderProgressBar/ContentByExecutionState.tsx

```diff
@@ -155,16 +180,22 @@ function ContentByExecutionState(props: ExecutionStateProps) {
                 </StatusMsg>
               </StatusWrapper>
               <StatusGraph>
-                <img src={cowMeditatingGraph} alt="Cow meditating ..." className="meditating-cow" />
-                {/*<p>
-                  Current price: <strong>$1200.56</strong>
-                </p>
-                <p>
-                  Your price: $1300.55 (<span>+8%</span>)
-                </p>*/}
-                <p>
-                  <strong>CoW Swap</strong> won&apos;t charge you if the trade is reverted{textAllowToCancel}.
-                </p>
+                <UnfillableMsgWrapper>
+                  <div>
+                    <p>
+                      Market is giving: <TextAmount strong>{amountsForMarketPriceDiff.currentPrice}</TextAmount>
```

Comment:
Is this text part of any design or is already in place?
If not I would prefer

```suggestion
                      Current market price: <TextAmount strong>{amountsForMarketPriceDiff.currentPrice}</TextAmount>
```

Reply:
Done!